schedule/yast/minimal+role_minimal/minimal+role_minimal_pvm.yaml

```diff
@@ -0,0 +1,31 @@
+---
+name: minimal+role_minimal
+description: >
+  SLE 15 specific registered installation with minimal role selected on PowerVM.
+  Installation is validated by successful boot and that YaST does not report
+  any issue.
+vars:
+  BSC1167736: '1'
+  SYSTEM_ROLE: minimal
+schedule:
+  - installation/bootloader_start
+  - installation/welcome
+  - installation/accept_license
+  - installation/scc_registration
+  - installation/addon_products_sle
+  - installation/system_role
+  - installation/partitioning
+  - installation/partitioning_finish
+  - installation/installer_timezone
+  - installation/user_settings
+  - installation/user_settings_root
+  - installation/installation_overview
+  - installation/edit_optional_kernel_cmd_parameters
```

Comment:
As you didn't define `OPT_KERNEL_PARAMS`, so it doesn't make sense to schedule this module. Without this module, also feel free to unify both schedules.

Reply:
Removed the extra module and unified schedule.